schedule/templates/login.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
Would be nice to also inherit it from base.html(or master.html in your case). You may want some scripts or logic for each page(for instance Google Analytics) and with base.html you have only one place to add it.